android/src/main/java/com/swmansion/rnscreens/SearchBarManager.kt

```diff
@@ -131,4 +132,56 @@ class SearchBarManager : ViewGroupManager<SearchBarView>() {
     companion object {
         const val REACT_CLASS = "RNSSearchBar"
     }
+
+    private fun logNotAvailable(propName: String) {
+        Log.w("RN SCREENS", "$propName prop is not available on Android")
+    }
+
+    // NativeCommands
+
+    override fun blur(view: SearchBarView?) {
+        view?.handleBlurJsRequest()
+    }
+
+    override fun focus(view: SearchBarView?) {
+        view?.handleFocusJsRequest()
+    }
+
+    override fun clearText(view: SearchBarView?) {
+        view?.handleClearTextJsRequest()
+    }
+
+    override fun toggleCancelButton(view: SearchBarView?, flag: Boolean) {
+        view?.handleToggleCancelButtonJsRequest(flag)
+    }
+
+    override fun setText(view: SearchBarView?, text: String?) {
+        view?.handleSetTextJsRequest(text)
+    }
+
+    override fun cancelSearch(view: SearchBarView?) {
+        view?.handleFocusJsRequest()
+    }
```

Comment:
Let's make sure that this approach with utilising generated `receiveCommand` implementation will still work on earlier React Native versions. Allegedly we support back to 0.68 but testing on 0.71 will be fine.

Reply:
I tested in on clean project with RN 0.71.0, imperative api seems to be working fine.